spec/acceptance/nodesets/docker/ubuntu-12.04.yml

```diff
@@ -12,8 +12,6 @@ HOSTS:
     docker_image_commands:
       - 'apt-get install -y net-tools wget'
       - 'locale-gen en_US.UTF-8'
-      - 'ln -s /opt/puppetlabs/bin/facter /usr/bin/facter'
```

Comment:
@vinzent I think depending on the version of beaker used, the path doesn't get exported and it's needed. Let me check.